codegen/lib/graphql_swift_gen/templates/type.swift.erb

```diff
@@ -115,82 +136,28 @@ extension <%= schema_name %> {
       <% end %>
 
       <% fields.each do |field| %>
-        <%= swift_attributes(field) %>
-        open var <%= escape_reserved_word(field.camelize_name) %>: <%= swift_output_type(field.type) %> {
+        <%= swift_doc(field, false) %>
+        <%= swift_attributes(field) %>open var <%= escape_reserved_word(field.camelize_name) %>: <%= swift_output_type(field.type) %> {
           return internalGet<%= field.classify_name %>()
         }
 
         <% unless field.args.empty? %>
           <%= swift_attributes(field) %>
-          open func aliased<%= field.classify_name %>(aliasSuffix: String) -> <%= swift_output_type(field.type) %> {
-            return internalGet<%= field.classify_name %>(aliasSuffix: aliasSuffix)
+          open func aliased<%= field.classify_name %>(alias: String) -> <%= swift_output_type(field.type) %> {
+            return internalGet<%= field.classify_name %>(alias: alias)
           }
         <% end %>
 
-        func internalGet<%= field.classify_name %>(aliasSuffix: String? = nil) -> <%= swift_output_type(field.type) %> {
-          return field(field: "<%= field.name %>", aliasSuffix: aliasSuffix) as! <%= swift_output_type(field.type) %>
+        func internalGet<%= field.classify_name %>(alias: String? = nil) -> <%= swift_output_type(field.type) %> {
+          return field(field: "<%= field.name %>", aliasSuffix: alias) as! <%= swift_output_type(field.type) %>
         }
       <% end %>
 
-      override open func childObjectType(key: String) -> GraphQL.ChildObjectType {
```

Comment:
These functions are used by the "Relay" layer of Shopify's ios app, so you can't remove them without breaking it right now.